cmd/cloud-controller-manager/app/options/options.go

```diff
@@ -45,7 +45,7 @@ import (
 	"k8s.io/kubernetes/pkg/controller"
 
 	// add the kubernetes feature gates
-	_ "k8s.io/kubernetes/pkg/features"
+	_ "k8s.io/apiserver/pkg/features"
```

Comment:
I don't think apiserver has all the feature gates we need to use from the cloud controller manager https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/features/kube_features.go#L162-L179

Reply:
For example: ServiceNodeExclusion and LegacyNodeRoleBehavior https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cloud-provider/controllers/service/controller.go#L72-L80

Reply:
Good catch!
https://github.com/kubernetes/kubernetes/blob/3a51aaf8b4baff1242ea30b08d599d913fe7a127/staging/src/k8s.io/cloud-provider/controllers/service/controller.go#L150, shows tge features usage you mentioned.
We need to move these features into https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/features/kube_features.go as part of this PR.

Reply:
Lets also do another review of the features the CCM uses.

Reply:
Hey Andrew, Thanks for the comments. Just curious for feature like ServiceNodeExclusion, it seems like to be removed from /pkg/features/kube_features already and be specified by `https://github.com/kubernetes/kubernetes/blob/c90427d42e8fe9ef37b61927afd2e12697c2d8a5/staging/src/k8s.io/cloud-provider/controllers/service/controller.go#L151`. Please correct me if my understanding is not correct🙏

Reply:
The feature gate name is copied to the controller (which is correct), but the feature gate registration which is then read by components is still in `pkg/features`.

Reply:
Have added the features tok8s.io/apiserver/pkg/features